src/mlpack/methods/ann/layer/celu.hpp

```diff
@@ -0,0 +1,203 @@
+/**
+ * @file celu.hpp
+ * @author Gaurav Singh
+ *
+ * Definition of the CELU activation function as descibed by Jonathan T. Barron.
+ *
+ * For more information, read the following paper.
+ *
+ * @code
+ * @article{
+ *   author  = {Jonathan T. Barron},
+ *   title   = {Continuously Differentiable Exponential Linear Units},
+ *   year    = {2017}
+ * }
+ * @endcode
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_CELU_HPP
+#define MLPACK_METHODS_ANN_LAYER_CELU_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The CELU activation function, defined by
+ *
+ * @f{eqnarray*}{
+ * f(x) &=& \left\{
+ *   \begin{array}{lr}
+ *    x & : x \ge 0 \\
+ *    \alpha(e^(\frac{x}{\alpha}) - 1) & : x < 0
+ *   \end{array}
+ * \right. \\
+ * f'(x) &=& \left\{
+ *   \begin{array}{lr}
+ *     1 & : x \ge 0 \\
+ *     (\frac{f(x)}{\alpha}) + 1 & : x < 0
+ *   \end{array}
+ * \right.
+ * @f}
+ *
+ * In the deterministic mode, there is no computation of the derivative.
+ *
+ * @note During training deterministic should be set to false and during
+ *       testing/inference deterministic should be set to true.
+ *
+ * @ tparam InputDataType Type of the input data (arma::colvec, arma::mat,
+ *         arma::sp_mat or arma::cube).
+ * @ tparam OutputDataType Type of the output data (arma::colvec, arma::mat,
```

Comment:
Hi @gaurav-singh1998, there is no space between @ tparam, I had to type it above like this otherwise a user gets mentioned.